src/Parser/OrientationParser.js

```diff
@@ -0,0 +1,110 @@
+import * as THREE from 'three';
```

Comment:
Global comment on `OrientationParser` : **very good optimization of calculations**